client_sdk/avalon_client_sdk/ethereum/ethereum_worker_registry_list_client.py

```diff
@@ -160,7 +178,26 @@ def registry_lookup(self, app_type_id=None):
                 lookupResult = \
                     self.__contract_instance.functions.registryLookUp(b"") \
                     .call()
-            return lookupResult
+            wCount, lookUpString, wList = lookupResult
```

Comment:
Please use PEP8 naming convention for variables. eg: lookupString -> lookup_string, lookupResult -> lookup_result